deploy/bin/tag-edge-endpoint-release.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Put a specific tag on an existing image in ECR
+# Assumptions:
+# - The image is already built and pushed to ECR
+# - The image is tagged with the git commit hash
+
+set -e  # Exit immediately on error
+set -o pipefail
+
+ECR_ACCOUNT=${ECR_ACCOUNT:-767397850842}
+ECR_REGION=${ECR_REGION:-us-west-2}
+
+# Ensure that you're in the same directory as this script before running it
+cd "$(dirname "$0")"
+
+# Check if an argument is provided
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <new-tag>"
+    exit 1
+fi
+
+NEW_TAG=$1
+
+# Only the pipeline can create releases
+if [[ "$NEW_TAG" == "pre-release" || "$NEW_TAG" == "release" ]]; then
```

Comment:
Should we add "latest" here as well, for now?

Reply:
Yes. I'll make that change.